Bugfix/scrubber digits decimal detection

jlobes opened the pull request:
# Decimal Detection in `scrubber.py`
## Fixes to `scrubber.py` and `test_scrubber.py`:
* changed `get_remove_digits_map` to incorporate a regex which also accounts for decimals numbers when **Remove Digits** is ticked, also had to change around the test suite function in `test_scrubber.py` that accounted for it.

mleblanc321 commented on lexos/processors/prepare/scrubber.py:
```diff
@@ -461,32 +461,25 @@ def get_remove_punctuation_map(
+    # any digits, this pattern will match:
+    # 1) ***
+    # 2) ***.***
+    pattern = re.compile(r"(\d)|((?<=\d)[\u0027|\u002C|\u002E|\u00B7"
```
we should capture as much as possible, so i think we want (\d**+**) [add + for runs of at least one (unicode) digit ... as well at the end of the regex \d**+**)

chantisnake commented:
I think removal map would not work in this case, because we are matching complicated regex, not just removing fixed strings, (like a dash)

Also, please use verbose regex, this regex is not that readable.

jlobes commented:
## Final Changes:
* changed the digits regex in `scrubber.py` to optionally accept signed numbers
* edited the unit test in `test_scrubber.py` accordingly
* documented the `get_remove_digits` function to properly describe the new way that we handle digits
* updated the test suite accordingly, adding two new files: `new_regex_jun19.txt` and `new_regex_jun19_Results.txt`
